libsel4/sel4_plat_include/imx8mp-evk/sel4/plat/api/constants.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020, Data61, CSIRO (ABN 41 687 119 230)
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
```

Comment:
Isn't all this SOC specific and not board specific? So we want to simply include a common imx8 constants.h here instead of allö the redundant definitions?

Reply:
If you want, you can follow what the imx8mm-evk did and symlink the include directory from imx8mm-evk.